docs/source/operations/projections/lccm.rst

```diff
@@ -0,0 +1,39 @@
+.. _lccm:
+
+********************************************************************************
+Lambert Conformal Conic (2SP Michigan)
+********************************************************************************
+
+.. figure:: ./images/lcc.png
+   :width: 500 px
+   :align: center
+   :alt:   Lambert Conformal Conic (2SP Michigan)
+
+   proj-string: ``+proj=lccm +lon_0=-90 +scale=1.0000382``
+
```

Comment:
Please write a short description of the projection here. If possible it would be nice to have a reference to an academic paper that describes the projection. Otherwise a reference the EPSG guidance note 7-2 or Wikipedia will do as well.

Reply:
I'll do that.

Reply:
Done in PJ_lcc.rst